irr/src/CIrrDeviceSDL.cpp

```diff
@@ -700,12 +700,11 @@ bool CIrrDeviceSDL::run()
 {
 	os::Timer::tick();
 
-	SEvent irrevent;
 	SDL_Event SDL_event;
 
 	while (!Close && wrap_PollEvent(&SDL_event)) {
+		SEvent irrevent{};
 		// os::Printer::log("event: ", core::stringc((int)SDL_event.type).c_str(),   ELL_INFORMATION);	// just for debugging
-
```

Comment:
It's a good idea not to make changes unrelated to the commit. I don't understand why this line was removed.